src/Break.tsx

```diff
@@ -17,6 +17,6 @@ export const Break = styled<IBreakProps, 'hr'>(Box as any)(thickness);
 Break.defaultProps = {
   as: 'hr',
   thickness: 1,
-  m: 'none',
-  borderColor: 'break.border',
+  m: '@none',
```

Comment:
Don't like this. `@none` should be `0` in the case of margin, but `none` in the case of border or shadow etc. Thus, this is ambiguous. Perhaps just `0`? Do we need theme variables representing none at all?